packages/reqresp/src/messages/Goodbye.ts

```diff
@@ -12,5 +13,13 @@ export const Goodbye: ProtocolDefinitionGenerator<phase0.Goodbye, phase0.Goodbye
     responseType: () => ssz.phase0.Goodbye,
     renderRequestBody: (req) => req.toString(10),
     contextBytes: {type: ContextBytesType.Empty},
+    inboundRateLimits: {
+      /**
+       * Metadata is part of handshake process, so we keep it
+       * equivalent to status message
+       */
+      byPeer: {quota: 2, quotaTime: minutes(1)},
```

Comment:
This is not metadata, but Ping. 2 / minute is very restrictive. What do other clients use?